core/src/main/resources/jenkins/monitor/EndOfLifeOperatingSystemAdminMonitor/message.properties

```
@@ -0,0 +1,9 @@
+End_of_Life_Heading={0} in Jenkins
+Upcoming_End_of_Life=<p class="jenkins-!-margin-bottom-0">You are running Jenkins on {0}.&nbsp;&nbsp;\
```

generally non breaking spaces aren't recommended as they can't be styled and can cause weird issues.
```suggestion
Upcoming_End_of_Life=<p class="jenkins-!-margin-bottom-0">You are running Jenkins on {0}. \
```

---

Happy to remove the non-breaking spaces.  I didn't like the layout without the non-breaking spaces because the text seemed to remove all space between the full stop and the first word of the following sentence.  I'll remove it to address your concern and capture a new screen shot for further discussion.